app.py

```diff
@@ -207,6 +209,28 @@ def save_identity():
     return jsonify({})
 
 
+# End Point example which verifies payload signature
+@app.route('/v1/signed_payload', methods=['POST'])
```

Comment:
You could introduce `/v1/identity` endpoint, which would print my authorized account:
```
{
    "id": "asdfs234674asfsd"
}
```